src/components/ClassEnrollmentConfirm.tsx

```diff
@@ -0,0 +1,406 @@
+import React, { SetStateAction } from "react";
+import {
+    Box,
+    Button,
+    HStack,
+    VStack,
+    Heading,
+    Text,
+    Stack,
+    Flex,
+} from "@chakra-ui/react";
+import { BackButton } from "@components/BackButton";
+import { CloseButton } from "@components/CloseButton";
+
+type ClassEnrollmentConfirmationProps = {
+    studentData: any; // Not actually any but I don't want to type the entire student object, let me know if I have to do this or if it's done somewhere else that can be used here
+    parentData: any; // ^^^ Same idea
```

Comment:
Smart! I think also just use the types that the prisma client generated for us, for student data, you could just use `Student from @prisma/client` and you can use User type from parentData (then for this above, you can just pass in the raw userData into this prop). Pretty neat huh.